app/controllers/UserController.java

```diff
@@ -194,7 +194,7 @@ public Result search(String search_string) {
         List<User> users = null;

```

Comment:
,am&e.$??%22?wwmnnmff,RX,
Fcttthhhhrrhrhhrhhrrhrhrhhrrhrehrhrhhhhrhhhrhhhhhhhrhhhghghhhhhhrhrhhghrrrrrrhrrrrrrrrrrrrrrrrrrhrrhrrrrrrrrrrrrrrrrrrrrhrrrrrrrrrrrrtrrrfcccg,gg,f 
g,o.,lazy that AQ